qa/L0_sequence_batcher/sequence_batcher_test.py

```diff
@@ -2267,7 +2266,7 @@ def test_queue_delay_full_min_util(self):
                             model_name,
                             dtype,
                             1001,
-                            (4000, 3000),
+                            (6000, 3000),
```

Comment:
Why do you change the threshold?

Reply:
There is a performance drop from the custom legacy sequence model to the new sequence backend is the need to increase this threshold.

Reply:
That is a concerning drop, from 4000ms to 6000ms (4 s to 6 s). I don't think the performance drop in sequence model change can be this dramatic, but in either way, we will likely want to fix this. Can you file a ticket for that and leave the original threshold values as inline comments?

Reply:
Done